app/views/api-docs/examples/index.html.md

```diff
@@ -20,6 +20,10 @@ On this page you may learn about how to:
 * [Sign a manifest](#sign-a-manifest)
 * [Management method codes](#management-method-codes)
 
+## Get an API Key
+If you are a developer from a TSDF and you'd like to request an API key so you can use the e-Manifest APIs, please check your organization's policies on implementing e-manifest.  Some companies may use unique API keys for each facility, others may use one API key for an entire company.  If you have questions please email emanifest@epa.gov (mailto:"emanifest@epa.gov).
```

Comment:
Did you want the `mailto` to be a hotlink? I think it will do that automatically without the explicit `mailto`.